package.xml

```diff
@@ -1,35 +1,40 @@
 <?xml version="1.0"?>
 <package format="3">
-  <name>bluespace_ai_xsens_mti_driver</name>
-  <version>1.0.1</version>
-  <description>ROS 2.0 driver for Xsens MTi IMU sensors</description>
-  <maintainer email="jasprit@bluespace.ai">BlueSpace.ai</maintainer>
-  <license>BSD</license>
+    <name>ros2_xsens_mti_driver</name>
+    <version>1.0.2</version>
+    <description>ROS 2.0 Humble driver for Xsens MTi IMU sensors</description>
+    <maintainer email="michal.bednarek@fourpoint.space">Michał Bednarek</maintainer>
+    <maintainer email="michal.malachowski@fourpoint.space">Michał Małachowski</maintainer>
+    <license>TODO</license>
```

Comment:
What about the license of the original code, does it affect or enforce us to use the same (or other specific) type of license?

Reply:
I think we can do whatever we want with BSD, but I'm not a lawyer. However, since it is a closed fork, I'll revoke the original license. Once we have more guidelines about licenses we will change that.